src/QtUtils/EventLoopTest.cpp

```diff
@@ -130,53 +130,91 @@ TEST(EventLoop, reuseLoop) {
   ASSERT_FALSE(loop.isRunning());
 
   // 1. normal quit
-  QMetaObject::invokeMethod(
-      &loop,
-      [&]() {
-        ASSERT_TRUE(loop.isRunning());
-        loop.quit();
-      },
-      Qt::QueuedConnection);
   {
-    const auto result = loop.exec();
-    ASSERT_FALSE(result.has_error());
-    EXPECT_EQ(result.value(), 0);
+    QMetaObject::invokeMethod(
+        &loop,
+        [&]() {
+          ASSERT_TRUE(loop.isRunning());
+          loop.quit();
+        },
+        Qt::QueuedConnection);
+    {
+      const auto result = loop.exec();
+      ASSERT_FALSE(result.has_error());
+      EXPECT_EQ(result.value(), 0);
```

Comment:
In case you want to "modernize" this file, you could use `EXPECT_THAT(orbit_test::HasValue(0))`. But I also recognize that you did not actually touch this

Reply:
Yes. On it.

Reply:
Done.